Api/src/controllers/officeController.js

```diff
@@ -0,0 +1,120 @@
+// Leaving this code for future reference on unique ID generation
+// import {
+//   v4 as uuidv4
+// } from 'uuid';
+
+import fs from 'fs';
+
+import data from "../data.json" assert { type: "json" };
+
+let offices = [];
+
+export const getOffices = (req, res) => {
+  res.send({
+    status: 200,
+    data: data.Offices
+  });
+
+  // Leaving this for reference on checking object keys
+  // console.log(Object.keys(data));
+};
+
+export const createOffice = (req, res, err) => {
+  const {
+    id,
+    type,
+    name
+  } = req.body;
+
+  const newOffice = {
+    id,
+    type,
+    name
+  };
+
+  // Leaving this for creating unique office IDs
+  // const officeId = uuidv4();
+
+  fs.writeFile('../data.json', JSON.stringify(newOffice), err => {
+    if (!err) {
+      res.send({
+        status: 404,
+        message: err
+      });
+    }
+    else {
+      return({
+        status: 200,
+        data: newOffice
+      })
+    }
+  });
+
+
+// This is commented because we are trying other methods of writing to the JSOn file
+  // offices.push(office);
```

Comment:
I'm trying out the `fs` because while testing out the `Post request` with Postman, I get a success message, however, there's no change to the `data.json file`.